File: 이건희/level1/소수 만들기.py
from itertools import combinations

def solution(nums):
    result = list(combinations(nums, 3))
    rslt = []

    for i in range(len(result)):
        rslt.append(result[i][0] + result[i][1] + result[i][2])

    cnt = 0

    for i in rslt:
        num = 0
        for j in range(2, i):
            if i % j == 0:
                num += 1
        if num == 0:
            cnt += 1

    return cnt

File: 이훈일/Programmers/Level1/소수 만들기.java
class Solution {
    public int solution(int[] nums) {
        int count = 0;
        for(int i = 0; i < nums.length - 2; i++) {
            for(int j = i+1; j < nums.length - 1; j++) {
                for(int k = j+1; k < nums.length; k++) {
                    int sum = nums[i] + nums[j] + nums[k];
                    int a = 0;
                    for(int num = 2; num <= sum/2; num++) {
                        if(sum % num == 0) {
                            a += 1;
                            break;
                        }
                    }
                    if(a == 0) {
                        count += 1;
                    }
                }
            }
        }

        return count;
    }
}
